M2L1HW/M2L1HW/Logger.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace M2L1HW
+{
+    /// <summary>
+    /// Singletone pattern is realized.
+    /// </summary>
+    public sealed class Logger
+    {
+        private static Logger instance = null;
+
+        private Logger()
+        {
+        }
+
+        public static Logger Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new Logger();
+                }
+
+                return instance;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets (idn why the ruleset want me to write that). List of all the logs generated in the program.
+        /// </summary>
+#pragma warning disable SA1609 // Property documentation should have value
+        public List<string> Logs { get; set; } = new List<string>();
```

Comment:
Whats List? ))
Even so u dont want to use string array, u can check StringBuilder